tests/unit/OrgPerson.spec.ts

```diff
@@ -542,114 +567,591 @@ describe('Org/Person component for a Correction filing', () => {
+        activeIndex: 0,
+        currentCompletingParty: null
+      }
+    })
+
+    expect(wrapper.find('.add-person-header').exists()).toBe(false)
+    expect(wrapper.find('.add-org-header').text()).toBe('Edit Business or Corporation')
+    expect(wrapper.find('.person-name label').exists()).toBe(false)
+    expect(wrapper.find('.confirm-name-change').exists()).toBe(false)
+    expect(wrapper.find('.edit-business-number .sub-header').text()).toBe('Business Number')
+    expect(wrapper.find('.show-business-number').exists()).toBe(false)
+    expect(wrapper.find('.org-look-up').exists()).toBe(false)
+    expect(wrapper.find('.org-manual-entry').exists()).toBe(false)
+    expect(wrapper.find('.other-edit-org .sub-header').text()).toBe('Business or Corporation Name')
+    expect(wrapper.find('.incorporation-number').exists()).toBe(false)
+    expect(wrapper.find('.email-address .sub-header').text()).toBe('Email Address')
+    expect(wrapper.find('#proprietor-partner-email').element['value']).toBe('proprietor@example.com')
+    expect(wrapper.find('.roles').exists()).toBe(false)
+    expect(wrapper.find('.mailing-address .sub-header').text()).toBe('Mailing Address')
+    expect(wrapper.find('.delivery-address .sub-header').text()).toBe('Delivery Address')
+    expect(wrapper.find(removeButtonSelector).exists()).toBe(true)
+
+    wrapper.destroy()
+  })
+
+  it('Changes existing GP partner - person', () => {
+    store.state.stateModel.tombstone.entityType = 'GP'
+    store.state.resourceModel = ChangeSolePropResource
+
+    const wrapper = mount(OrgPerson, {
+      store,
+      vuetify,
+      propsData: {
+        currentOrgPerson: PartnerPersonData,
+        activeIndex: 0,
+        currentCompletingParty: null
+      }
+    })
+
+    expect(wrapper.find('.add-person-header').text()).toBe('Edit Person')
+    expect(wrapper.find('.add-org-header').exists()).toBe(false)
+    expect(wrapper.find('.person-name label').text()).toBe('Person\'s Name')
+    expect(wrapper.find('.confirm-name-change').exists()).toBe(false)
+    expect(wrapper.find('.edit-business-number').exists()).toBe(false)
+    expect(wrapper.find('.show-business-number').exists()).toBe(false)
+    expect(wrapper.find('.org-look-up').exists()).toBe(false)
+    expect(wrapper.find('.org-manual-entry').exists()).toBe(false)
+    expect(wrapper.find('.other-edit-org').exists()).toBe(false)
+    expect(wrapper.find('.incorporation-number').exists()).toBe(false)
+    expect(wrapper.find('.email-address .sub-header').text()).toBe('Email Address')
+    expect(wrapper.find('#proprietor-partner-email').element['value']).toBe('partner@example.com')
+    expect(wrapper.find('.roles').exists()).toBe(false)
+    expect(wrapper.find('.mailing-address .sub-header').text()).toBe('Mailing Address')
+    expect(wrapper.find('.delivery-address .sub-header').text()).toBe('Delivery Address')
+    expect(wrapper.find(removeButtonSelector).exists()).toBe(true)
+
+    wrapper.destroy()
```

Comment:
Yes, but in that case I would prefer the mount to be in a beforeEach() method, for symmetry. If nothing comes up today then I will look at cleaning this up a bit.

Reply:
Right on, ya I was just curious if there was a reason you weren't doing it in these ones. I'll make sure not to merge incase you add those changes.

Reply:
I didn't do here because:
- I wasn't sure if I had time to do it
- in some cases I prefer individual tests -- they're less complex so easier to understand (and I did code them sequentially, not all at once)

In this case, with so much commonality between them, they are great candidates to parameterize.

Reply:
Right, I thought maybe since not all of your tests used `wrapper` that maybe it was a resource saving measure to prevent initializing it for the tests where it is not used.

Reply:
See test results in comment above.